src/cmd/api/db/dao/video.go

```diff
@@ -0,0 +1,62 @@
+package dao
+
+import (
+	"database/sql"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/Sogilis/Voogle/src/cmd/api/db/models"
+)
+
+func PutVideo(db *sql.DB, video models.VideoModelUpload) (string, error) {
+	id := uuid.NewString()
+	clientId := uuid.NewString()
+
+	query := "INSERT INTO videos (id, client_id, title) VALUES ('" + id + "','" + clientId + "', ?);"
+	res, err := db.Exec(query, video.Title)
+	if err != nil {
+		log.Error("Error while insert into videos:", err)
+		return "", err
+	}
+
+	nbRowAff, err := res.RowsAffected()
+	if err != nil {
+		log.Error("Error, can't know how many rows affected:", err)
+		return "", err
+	}
+
+	log.Infof("%d row(s) inserted", nbRowAff)
+	return clientId, nil
+}
+
+func GetVideos(db *sql.DB) ([]models.VideoModel, error) {
```

Comment:
Later, we will probably want to limit the quantity of listed video or paginate them. If we don't do it now, we should add an issue about it.

Reply:
This issue ? #157